packages/shared/styles/components/molecules/SfBar.scss

```diff
@@ -3,21 +3,21 @@
 .sf-bar {
   box-sizing: border-box;
   display: flex;
-  justify-content: space-between;
   align-items: center;
   height: var(--bar-height, 3.125rem);
-  padding: var(--bar-padding, 0 var(--spacer-big));
-  background-color: var(--bar-background, var(--c-light));
-  font: var(
+  padding: var(--bar-padding, 0 var(--spacer-regular));
+  background-color: var(--bar-background, #f2f2f2);
```

Comment:
Like we discussed -  please ask Marcin and change to one of the colors we already declared